packages/modules/src/modules/epics/checkout/ContributionsEpicCheckout.tsx

```diff
@@ -0,0 +1,79 @@
+import { css } from '@emotion/react';
+import { addRegionIdAndTrackingParamsToSupportUrl } from '@sdc/shared/lib';
+import { Tracking } from '@sdc/shared/src/types';
+import React, { RefObject } from 'react';
+
+const styles = {
+    iframeContainer: css`
+        border: none;
+        width: 100%;
+    `,
+};
+
+// Currently components don't know if they're in DEV/CODE/PROD
+// hence why we hardcode urls to PROD. For convenience for future
+// dev work we've left in the CODE and DEV urls below.
```

Comment:
It would be great to get some config in place to handle this as it's a production bug waiting to happen but not for this PR I guess

Reply:
Yeah very much agree. I'm not sure exactly how that would look (maybe the platform would provide the stage through the props?). But it would affect a few places (eg reminders too) so would make sense as a separate piece of work

Reply:
The epic component does have the stage:
https://github.com/guardian/support-dotcom-components/blob/main/packages/shared/src/types/props/epic.ts#L30

E.g. DCR passes it in here: https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/web
/components/SlotBodyEnd/ReaderRevenueEpic.tsx#L196

Reply:
oh that's great! I didn't realise! I'll make a follow up PR :)